policy/modules/kernel/kernel.if

```diff
@@ -1392,6 +1392,7 @@ interface(`kernel_read_core_if',`
 	')
 
 	allow $1 self:capability sys_rawio;
+	allow $1 self:lockdown confidentiality;
```

Comment:
Could you split this out into a separate commit? (It doesn't seem directly related to the unsigned module loading.)